### PR TITLE
Setup initial tests for the master component HomeValueApp

### PR DESCRIPTION
**Pull request by phuctranlv**
@dontpressthat , @richard-c-gill , @dhkaneda 
Can you guys please review and approve?

### CODE REVIEW

File: __tests__/HomeValueApp.test.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  sinon.spy(HomeValueApp.prototype, 'componentDidMount');
+  const wrapper = mount(<HomeValueApp />);
+  const spyCall = $.ajax.getCall(0);
+
+  it('The componentDidMount was called', () => {
+    console.log('Test: The componentDidMount was called.');
+    expect(HomeValueApp.prototype.componentDidMount).toHaveProperty('callCount', 1);
+  });
+
+  it('The initial ajax request was made', () => {
```
Comment (richard-c-gill) [inline __tests__/HomeValueApp.test.jsx:19]: Still using ajax huh?...

File: client/components/HomeValueApp.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+class HomeValueApp extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      homeSummary: 'request home summary',
+    };
+  }
+
+  componentDidMount() {
+    $.ajax({
```
Comment (richard-c-gill) [inline client/components/HomeValueApp.jsx:13]: Tisk tisk

File: client/index.jsx [hunk 3]
```diff
@@ -1,7 +1,8 @@
 // eslint-disable-next-line import/extensions
-import React from 'react';
-import ReactDOM from 'react-dom';
```
Comment (richard-c-gill) [inline client/index.jsx:3]: wtf ES lint doesn't let you use ES6 syntax?

File: package.json [hunk 4]
```diff
@@ -48,9 +53,8 @@
     "babel-cli": "^6.26.0",
     "express": "^4.17.1",
     "faker": "^4.1.0",
+    "jquery": "^3.4.1",
     "mysql": "^2.18.1",
```
Comment (richard-c-gill) [inline package.json:57]: Thought you were using mongo... you should download the mysql2 version

**Review by richard-c-gill — APPROVED** [review]
Meh

File: client/index.jsx (hunk 3 above)
Comment (phuctranlv) [reply client/index.jsx:3]: I was stuck on this for like 2 hours... I kept getting errors from using import-from syntax

**Pull request merged**